### PR TITLE
[0.3.1] Functionality to view Game Stage and able to click on enemy Board

### PR DESCRIPTION
**Pull request by AdrianHarenczyk**
This merge request is there to complete assignment of getting 0.3.1 version live, which was to make a simple mock-like view prepared to playable stage. In this view player can push buttons on enemy board, which then appear with dot indicating that it's a miss. Later on in newer versions functionality to transfer enemy fleet behind the scenes will be added. In this version the focus is on preparation of ground to work on further down the road. 

**Please check any clean code violations and such on.**

### CODE REVIEW

File: client/src/main/java/gui/FleetDAO.java [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 /**
  * Layer of abstraction over fleet generation.
- * De facto functional interface.
+ * In fact functional interface.
  */
-public interface FleetDAO {
-    GUIFleet getGUIFleet();
+interface FleetDAO {
```
Comment (tomeks86) [inline client/src/main/java/gui/FleetDAO.java:7]: DAO to kojarzy się z bazą danych...

**Review by tomeks86 — APPROVED** [review]
Nie wiem czy macie już wszystkie zaplanowane na nową wersję funkcjonalności, żeby podbić numer wersji.

File: client/src/main/java/gui/FleetDAO.java (hunk 1 above)
Comment (krzysieksr) [reply client/src/main/java/gui/FleetDAO.java:7]: :+1:

File: client/src/main/java/gui/MainStageController.java [hunk 2]
```diff
@@ -124,5 +107,13 @@ private Runnable addToTextArea(String message) {
         return () -> outputFromServer.appendText(message + NEW_LINE);
     }
 
+    private void disableStartWhenNoFleet() {
+        boolean fleetDoesNotExist = (fleet == null);
```
Comment (krzysieksr) [inline client/src/main/java/gui/MainStageController.java:111]: break this statement into 2 separate boolean values.

**Review by krzysieksr — APPROVED** [review]
Make minor fixes and you can push it.

**Pull request merged**